controls/anssi-minimal.yml

```diff
@@ -0,0 +1,61 @@
+policy: 'ANSSI-BP-028'
+title: 'ANSSI-BP-028'
+id: anssi
+version: '1.2'
+source: https://www.ssi.gouv.fr/uploads/2019/03/linux_configuration-en-v1.2.pdf
+controls:
+  - id: R1
+    title: Minimization of installed services
+    automated: partially  # The list of essential services is not objective.
```

Comment:
What about some `notes` key? Its contents could propagate to the generated outputs that we plan to introduce in future.